tests/test_uuid.nim

```diff
@@ -0,0 +1,140 @@
+import std/[unittest]
+import "."/[uuid/uuid]
+
+proc main =
```

Comment:
Maybe also a test here that the `uuid` function returns a valid `uuid`? I know it is duplicated in the `test_binary` file, but I feel like it should also be specified here. That other test is then more about that the right number of uuids is specified (with added validity checks).

Reply:
One problem is that I didn't write a UUID function that returns a string, but only one that writes to stdout (which is arguably a bad pattern).

I could add that proc and test it, but I don't know if that helps much - it doesn't test that `configlet uuid` actually uses that proc.

Reply:
I think I would indeed separate the generating of the UUID and the writing of it